### DISCUSSION TITLE
Monitoring for v2/v3/etc fixed releases on anime episodes

### DISCUSSION THREAD

**Discussion by deusxanime** [pymedusa/Medusa#10551] [Ideas & Feature requests]
There already exists a mechanism in Medusa to check for fixes/new versions of "regular" TV episodes using the PROPER and REPACK release tags. For anime releases, most anime subbing and release groups tend to use version tags for new or fixed releases, such as v2, v3, v4, etc (I think the highest I've seen is v6, but possible to go higher of course) which are put in after the episode number. Would it be possible to look for those and download fixed anime episodes in the same way PROPER/REPACK is done? Right now I have to manually monitor for and grab those releases, so it would nice to be automated right in Medusa.

Example of the "v2" tag in a normal anime release, which uses absolute episode numbering:
[DameDesuYo] Shingeki no Kyojin (The Final Season Part 2) - 76v2 (1920x1080 10bit AAC) [BBC25CF3].mkv

Uncommon to use the sXXeXX numbering in an anime release, but even so this group still stuck with the anime convention and used "v2" for a fixed episode:
Black Rock Shooter - Dawn Fall - S01E04v2 - 1080p WEB H.264 -NanDesuKa (DSNP).mkv

**Comment by p0psicles** [comment]
One thing that's worth to check out is if guessit supports this. And if not would be worth doing a feature request for that

**Comment by deusxanime** [comment]
Sorry, this got lost in my tabs and so I didn't follow up. I went over to guessit's github and was going to create an issue but first searched to see if it had been brought up before. It looks like they added support for it back as far as 2014!

https://github.com/guessit-io/guessit/issues/138

So perhaps it is already implemented?

**Reply by p0psicles** [reply]
I'll see if we can reuse the existing proper search for this
